cmake/dependencies.cmake

```diff
@@ -52,9 +70,10 @@ if(NOT USE_SYSTEM_MKSQUASHFS)
 
     ExternalProject_Add(mksquashfs
         GIT_REPOSITORY https://github.com/plougher/squashfs-tools/
-        GIT_TAG 4.4
+        GIT_TAG 4.5.1
```

Comment:
Needed to get https://github.com/plougher/squashfs-tools/pull/83, so I just updated all the way to latest release.